Dockerfile

```diff
@@ -5,4 +5,4 @@ COPY package*.json ./
 RUN npm install
 COPY . .
```

Comment:
Can you add a `RUN mkdir /ants` under the copy line here? `copy . .` should in theory copy everything including the ants folder in root, but it wasn't locally and we want to make sure the ants folder is present for the volume mapping to work.

Reply:
Yep